recipe/meta.yaml

```diff
@@ -22,7 +22,8 @@ source:
 
 build:
   number: 0
-  skip: true  # [(not win64 and not linux64 and (ppc64le and cuda_compiler_version != "10.2") and not (aarch64 and arm_variant_type == "sbsa")) or cuda_compiler_version in (undefined, "None")]
+  ? skip__###conda-selector###__(not win64 and not linux64 and (ppc64le and cuda_compiler_version != "10.2") and not (aarch64 and arm_variant_type == "sbsa")) or cuda_compiler_version in (undefined, "None")
+  : true
```

Comment:
@conda-forge/bot Whenever the bot issues new-version PRs we always get this line messed up and I had to fix it manually. Any idea what could be the issue?

Reply:
Nope. Let's open an issue on the bot repo.

Reply:
@beckermr which repo? I forgot 😄

Reply:
regro/cf-scripts

Reply:
Done: https://github.com/regro/cf-scripts/issues/1499